### DISCUSSION TITLE
Eliminate the "Keep funds in bisq wallet" and "Move funds to External Wallet" button

### DISCUSSION THREAD

**Discussion by leo816** [bisq-network/bisq#5410] [Ideas]
On last week's support call we mentioned this topic:

After a trade is completed you get the option to "keep funds in your bisq wallet" but the reality is that the funds are automatically added to your wallet and the button serves no real purpose. The "Move funds to external wallet" does make sense but I see it as a confusing step that is not necessary. A user can just go into send funds and move them to his external wallet without an issue. Perhaps having just a notification saying your funds have been added to your wallet would be better in my eyes.
If there is another reason why this step exists that Im missing please let me know.

**Comment by pazza83** [comment]
Hi @leo816 

I think the Keep funds in bisq wallet screen is good for UI but the wording should be changed to make it more clear. I also think that there does not need to be the option to withdraw on this screen.

Currently the screen looks like this:

![keep funds](https://user-images.githubusercontent.com/66367547/115095475-69070a80-9f19-11eb-9bd0-dc80276dcdb5.png)

I would propose it to look like this:

![NEW keep funds](https://user-images.githubusercontent.com/66367547/115095489-6f958200-9f19-11eb-8266-4c3ece079a68.png)

**Reply by ghost** [reply]
For that button, `CLOSE OPEN TRADE` sounds a bit off, how about just `CLOSE`

I'd like to include this as part of https://github.com/bisq-network/bisq/pull/5870 because especially for trades that have gone to arbitration and lost, there may be 0 funds so having Keep / Withdraw buttons would not make sense.

**Comment by ghubstan** [comment]
> After a trade is completed you get the option to "keep funds in your bisq wallet" but the reality is that the funds are automatically added to your wallet and the button serves no real purpose. ...  If there is another reason why this step exists that Im missing please let me know.

Hi @jmacxx, I am pretty sure the UI's "keep funds" action is what moves an open trade into the closed trades list (and moves the trade from the open trades persistence file to the closed trades persistence file).  I had to get the API to perform this action with a `bisq-cli keepfunds` command, and this step is easy to overlook or forget.

However you decide to change this, please ping me here so I can adjust the API?

**Comment by Conza88** [comment]
A link to a wiki article describing everything a bit better? 

I recall being a total noob, and got confused by this. 

Didn't really get Bisq had its own wallet, on top of being an exchange. And felt I had to best practice, send to my own wallet elsewhere.. but that gets funds out of Bisq.
